conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -26,8 +26,13 @@ setup_conda_rc "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 
 {% if build_setup -%}
 {{ build_setup }}{% endif -%}
-{% if yum_build_setup is defined -%}
-{{ yum_build_setup }}{% endif -%}
+
+# If we have yum requirements build a new specialized image that we will use to run our build in that includes
+# those dependencies.
```

Comment:
Can you update this comment or remove it?

Reply:
Actually are we able to just revert all of the changes to this file now?

Reply:
we could, though this does add the advantage that we dont need a rerender if we've changed yum_requirements

Reply:
Yeah, this is roughly how I proposed `yum_requirements.txt` originally ( https://github.com/conda-forge/conda-smithy/pull/135 ), but Phil made a compelling argument that they should be listed explicitly ( https://github.com/conda-forge/conda-smithy/pull/134 ). Ultimately I don't feel strongly either way, but it seems the crux of the needed change is quite simple; so, it would be nice if this PR reflected that. Orthogonal changes are better left for other PRs IMHO.